src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanel.js

```diff
@@ -29,7 +29,7 @@ class DocumentPanel extends Component {
         <>
           <Media greaterThanOrEqual="tablet">
             <div
-              className="document-panel"
+              className="document-series-panel"
```

Comment:
is this intentional ? seems like a mismatch between naming of the component and the class